docs/modules/plugins/pages/plugin-mobitru.adoc

```diff
@@ -43,6 +48,43 @@ selenium.grid.capabilities.platformVersion=12
 selenium.grid.capabilities.deviceName=iPhone 13
 ----
 
+IMPORTANT: For search more specific device you can additionally with requested capabilities use
+device search parameters properties specified below. If the device name or udid was specified in the
+requested capabilities, then device search parameters properties will be ignored and device will be searched only using
+requested capabilities.
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[#mobitru_device_search_properties]
+[cols="6,2,5", options="header"]
+
+|===
+
+|Property Name
+|Acceptable values
+|Description
+
+|`*mobitru.device.type*`
```

Comment:
```suggestion
|`*selenium.grid.capabilities.mobitru\:device-type*`
|`*selenium.grid.capabilities.mobitru\:manufacturer*`
|`*selenium.grid.capabilities.mobitru\:model*`.  (is this the same as deviceName capability ?)
```

it will be a flexible mechanism thus caps supported not only at property level OOB

Reply:
Model is not the same as deviceName, but it is part of deviceName. Examples of model: '6s' or 'Pixel 6'

Reply:
So it's even more complicated :) You need to take into account that the user might use deviceName in the capabilities, I am not sure how mobitru will behave if you have both udid and deviceName (which might be different from acquired device) in capabilities